### PR TITLE
Fix master pipeline

### PR DESCRIPTION
**Pull request by Esysc**
```
ERRO[0000] could not parse additional tags: failed to read file at "/tmp/build/put/version/date.txt": open /tmp/build/put/version/date.txt: no such file or directory 
```

This error raised after merging, didn't catch this copy/paste leftover while working on the feature.

Besides that, and following the comments I received, I use the `semver` Concourse resource for adding additional tag, discarding the task `create config` that is not needed anymore.

### CODE REVIEW

**Comment by Pix4D-Janus** [comment]
pipeline_setter_bot: your pipeline is set: https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-fix-pipeline

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 1]
```diff
@@ -200,4 +201,3 @@ jobs:
   - put: pypi.docker
     params:
       image: pypi.dev-docker/image.tar
-      additional_tags: ((docker_tag_add_file))
```
Comment (aliculPix4D) [inline ci/pipelines/concourse-pypi-resource-pipeline-template.yml:203]: why? you want only latest? and each time to overwrite the latest?
isn't it better to use the version? you just forgot to stream the version data.

Comment (aliculPix4D) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:203]: add `inputs: [version]`

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 2]
```diff
@@ -161,6 +161,7 @@ jobs:
     inputs: [image, version]
     params:
       image: image/image.tar
+      additional_tags: ((docker_tag_add_file))
```
Comment (aliculPix4D) [inline ci/pipelines/concourse-pypi-resource-pipeline-template.yml:164]: are you sure?

Comment (Esysc) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:164]: Do you think it is useless there?

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 3]
```diff
@@ -191,13 +192,21 @@ jobs:
...
+  <<: *base-job
   plan:
+  - get: linux-builder.git
+  - get: pypi.git
   - get: pypi.dev-docker
     trigger: ((trigger))
     passed: [pypi-acceptance-test]
     params:
       format: oci
+  - task: config-create
```
[the remark this answers is not in the document]
Comment (Esysc) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:204]: In my understanding we need to create the `version` input for the `put` step.

[the remark this answers is not in the document]
Comment (aliculPix4D) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:202]: But you have the version from the version resource in format 0.21.0 or something. Just adapt the resource for feature/master workflow (artifacts/final) and use that file. No need to create a special one in YYYYMMDDHHMMSS format.

or just use `repo.git/.git/describe_ref`

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 4]
```diff
@@ -161,7 +161,6 @@ jobs:
     inputs: [image, version]
```
Comment (aliculPix4D) [inline ci/pipelines/concourse-pypi-resource-pipeline-template.yml:161]: you can remove version here. No need to stream if its unused.

Comment (Esysc) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml]: Absolutely, it's not used

File: ci/settings/master-branch.yml [hunk 5]
```diff
@@ -1,6 +1,8 @@
-project: concourse-pypi-resource-pipeline-template
+project: concourse-pypi-resource-pipeline
 docker_tag: latest
-docker_tag_add_file: version/date.txt
+docker_tag_add_file: version/version
+semver_initial_version: 0.9.0
+semver_key: artifacts/concourse-example-pypi/version
```
Comment (odormond) [inline ci/settings/master-branch.yml:5]: Shouldn't that be:
```suggestion
semver_key: final/concourse-example-pypi/version
```
as it's for the master branch it should be persisted but stuff in `artifacts` get pruned after some time.

Comment (Esysc) [reply ci/settings/master-branch.yml:5]: Ah I should change the name as I don't use anymore the concourse-example package. Also the `version` used for building the test wheel will not be bumped, just used to have a version. So finally this version is really used and bumped for `docker_additional_tags`. In this case I could use `artifact` for the feature pipeline and `final` for master.

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 6]
```diff
@@ -171,33 +160,35 @@ jobs:
...
       glob: 'dist/*'
 
 - name: publish-concourse-pypi-resource
+  <<: *base-job
   plan:
+  - get: pypi.git
+  - get: version
+    params:
+      bump: patch
   - get: pypi.dev-docker
```
Comment (aliculPix4D) [inline ci/pipelines/concourse-pypi-resource-pipeline-template.yml:183]: are we sure that we are using the image that was just produced in step before?

**Review by aliculPix4D — APPROVED** [review]

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml (hunk 6 above)
Comment (Esysc) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:183]: I'd say yes. The output of the job that creates the Docker image is the input of the acceptance test.
you can check for example:
- https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-fix-pipeline/jobs/concourse-pypi-resource-build/builds/9
- https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-fix-pipeline/jobs/pypi-acceptance-test/builds/9

and check the same sha is used:
```
sha256:55dccffbd2305fc40640cbd488a4a699f97272b9906facab77b5f4a38df8d771
```

**Review by odormond — APPROVED** [review]

**Pull request merged**